src/bahmni-helm/templates/apps/backup-services/cron-jobs.yml

```diff
@@ -0,0 +1,254 @@
+{{ if .Values.apps }}{{ if .Values.apps.backup_services }} {{ if .Values.apps.backup_services.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-mysql-backup
```

Comment:
`{{ .Release.Name }}-mysql-backup` seems like it should be named `{{ .Release.Name }}-openmrs-db-backup`